app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, filename, copy_filename = command.split()
+    if cp != "cp" and filename != copy_filename:
+        with open(filename, "r") as file_in, open(
+                copy_filename, "w") as file_out:
+            temp = file_in.read()
```

Comment:
just remove this redundant variable as Serhii said. You can use `file_in.read()` in the line below